tests/components/velbus/test_config_flow.py

```diff
@@ -0,0 +1,99 @@
+"""Tests for the Velbus config flow."""
+from unittest.mock import patch, Mock
+
+import pytest
+
+from homeassistant import data_entry_flow
+from homeassistant.components.velbus import config_flow
+from homeassistant.const import CONF_PORT, CONF_NAME
+
+from tests.common import MockConfigEntry
+
+PORT_SERIAL = '/dev/ttyACME100'
+PORT_TCP = '127.0.1.0.1:3788'
+
+
+@pytest.fixture(name='controller_assert')
+def mock_controller_assert():
+    """Mock the velbus controller with an assert."""
+    controller = Mock()
+    with patch('velbus.Controller', side_effect=Exception()):
+        yield controller
```

Comment:
We don't need to yield a mock instance since there will be an exception. Just `yield`.